backend/data/program-credentials/merge-credentials.py

```diff
@@ -160,6 +160,28 @@ def label_credential_type(row: pd.Series):
     if (row['LEADTOINDUSTRYCREDENTIAL'] == "1"):
         return CredentialType.Certification
 
+    # TODO: Track Down Lookup Values for Provider TYPEIDs.
```

Comment:
I'd say it's generally useful to track the TODO with an issue so it doesn't get lost later on. But if this is something that'll be on your radar in near future work, it's overkill.

Reply:
100% agree. I meant to create this GitHub issue and then add it to the comment. Will fix.